subprojects/emcee/worker/src/test/kotlin/com/avito/emcee/worker/rest/HttpServerTest.kt

```diff
@@ -56,15 +65,11 @@ class HttpServerTest {
         server.stop()
     }
 
-    private fun findAvailablePort(): Int {
-        val s = ServerSocket(0)
-        val port = s.localPort
-        s.close()
-        return port
+    private fun createServer(): HttpServer {
+        return HttpServer(
```

Comment:
it looks more pretty with builder pattern for me

Reply:
I against the builder in current case. In my opinion builder needs when you have many parameters and many configurations of object and you make them dynamically in runtime. It's not about current case